india_compliance/gst_india/utils/test_e_invoice.py

```diff
@@ -74,6 +74,41 @@ def test_request_data_for_different_shipping_dispatch_address(self):
             EInvoiceData(si).get_data(),
         )
 
+    @change_settings("GST Settings", {"enable_overseas_transactions": 1})
+    def test_request_data_for_foreign_transactions(self):
+        test_data = self.e_invoice_test_data.foreign_transaction
+        si = create_sales_invoice(
+            **test_data.get("kwargs"), qty=1000, do_not_submit=True
+        )
+        si.update(
+            {
+                "gst_category": "Overseas",
+                "shipping_bill_number": "1234",
+                "shipping_bill_date": frappe.utils.today(),
+                "port_code": "INABG1",
+            }
+        )
+
+        if si.customer_address != si.port_address:
```

Comment:
You should know your port address in the sales invoice.

Reply:
Already setting port address from test records.